src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -13,6 +13,7 @@
 			<![CDATA[
 New features:
 - Added support for using sever level rule in vulnerability assessment baseline operations.
+- Enable customers to upload their TDE certificate to CMS
```

Comment:
I published 1.17 today (on request from vulnerability assessment team), so now you need to bump this up to 1.18 and wipe out the old release notes (i.e. just line 15)

Reply:
Bump to 1.18 both in this file and in AssemblyInfo.cs

Reply:
Sure will do